From ad0af956f558f20e7678184dad79849062e87817 Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Thu, 21 Aug 2008 11:13:21 +0800
Subject: [PATCH] ENGR00088656 [V4L2]flush cache everytime by WB policy

The cache should be flush everytime when L1 cache using WB policy.

Signed-off-by: Jason Chen <b02280@freescale.com>
---
 drivers/media/video/mxc/output/mxc_v4l2_output.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/output/mxc_v4l2_output.c b/drivers/media/video/mxc/output/mxc_v4l2_output.c
index c26e0df..ca374c2 100644
--- a/drivers/media/video/mxc/output/mxc_v4l2_output.c
+++ b/drivers/media/video/mxc/output/mxc_v4l2_output.c
@@ -1262,7 +1262,7 @@ mxc_v4l2out_do_ioctl(struct inode *inode, struct file *file,
 
 			/* mmapped buffers are L1 WB cached,
 			 * so we need to clean them */
-			if (buf->flags & V4L2_BUF_FLAG_MAPPED) {
+			if (buf->memory & V4L2_MEMORY_MMAP) {
 				flush_cache_all();
 			}
 
-- 
1.5.4.4

